datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -64,6 +71,16 @@ def start(ctx, check, env, agent, dev, base, env_vars):
         echo_info('See what is available via `ddev env ls {}`.'.format(check))
         abort()
 
+    env_python_version = get_tox_env_python_version(env)
+    if not python:
+        # Make the tox environment Python specifier influence the Agent
+        python = env_python_version or DEFAULT_PYTHON_VERSION
+    elif env_python_version and env_python_version != str(python):
```

Comment:
Since `get_tox_env_python_version(env)` is returning an int, should we have this instead ?
```suggestion
    elif env_python_version and env_python_version != int(python):
```